pkg/rpm/salt-minion

```diff
@@ -25,12 +25,11 @@
 # processname: /usr/bin/salt-minion
 
 # Allow these to be overridden for tests
-: ${SALTMINION_BINDIR:=/usr/bin}
-: ${SALTMINION_SYSCONFDIR:=/etc}
-: ${SALTMINION_PYTHON:=python}    # Python must be 2.6 or newer
+: "${SALTMINION_BINDIR:=/usr/bin}"
+: "${SALTMINION_SYSCONFDIR:=/etc}"
 
 # Default values (can be overridden in settings file)
-: ${USER:=$(id -nu)}
+: "${USER:=$(id -nu)}"
```

Comment:
Please see my comment above. By the way, `SALTMINION_CONFIGS` variable also could be affected by pathname expansions, so I will fix it as well.